app/views/katello/api/v2/content_facet/show.json.rabl

```diff
@@ -0,0 +1,11 @@
+child :content_facet => :content_attributes do
```

Comment:
yeah this is the first time we're 'properly extending' an object.  so partially creating a convention.  Would:
content_details and subscription_details  sound better?   I really don't like _attributes so trying to think of something better :)